lib/stripe/charges.ex

```diff
@@ -309,11 +323,12 @@ defmodule Stripe.Charges do
       {:ok, charge} = Stripe.Charges.refund("charge_id", "my_key")
 
   """
-  def refund(id, key) do
-    Stripe.make_request_with_key(:post, "#{@endpoint}/#{id}/refunds", key)
+  def refund(id, key, params) do
```

Comment:
This makes the PR introduce a breaking change. Any application using this library which would call `refund/2` will now have to be fixed. Luckily, there's a simple way to avoid this problem:

```Elixir
def refund(id, key, params \\ %{}) do
```